### ISSUE TITLE
liveSearchNormalize considering SPAN closure

### ISSUE THREAD

**Issue by marceloudi** [snapappointments/bootstrap-select#1421]
Hi! I'm facing the following situation:

Setting `liveSearchNormalize: true`, and searching for "clas" word, the match process consider the `data-normalized-text` property, showing incorrect results caused by <span class="text"></span> closure outerHtml text.

![image](https://cloud.githubusercontent.com/assets/751071/16139637/ec5124a8-341f-11e6-9ebf-7728198ddf6b.png)

So, I found a dumb solution (I know that this approach is not "correct"): retrive only the innerHtml by a regex group match:

![image](https://cloud.githubusercontent.com/assets/751071/16139704/a09dffee-3420-11e6-8f0d-67b76df91af9.png)

This way I achieve correct results.
Is this a real bug or I misunderstood this property?

Regards!

**Comment by marceloudi** [comment]
Update: extract only text from html:

 ```
 function htmlEscape(html) {
	//First strip quotes, and get only text
	html = $(html.replace(/['"]+/g, '')).text();

    ...
}
```

**Comment by caseyjhol** [comment]
Fixed in v1.13.0.
